docs/metrics.md

```diff
@@ -0,0 +1,435 @@
+        "add_new_button",
+        "search_box"],
+    extra: {"fxa_uid": uid}
+  }
+});
+```
+To register the render events:
+```javascript
+Services.telemetry.registerEvents("item_list.render", {
+  "item_list.render": {
+    methods: ["render"],
+    objects: ["item_list_empty",
+        "item_list_populated"],
+    extra: {"fxa_uid": uid}
+  }
+});
+```
+
+
+To record (for example) a click on the add new entry button:
+```javascript
+Services.telemetry.recordEvent("item_list.interaction",
+                    "click",
+                    "add_new_button",
+                    null,
+                    {"fxa_uid": uid});
+```
+
+### Item View Interaction Events
+When a user clicks on an item from the item list, they will be directed to the item view. We will log clicks on each of the five buttons in this view (also the **objects** used in the event): `edit_entry_button`, `delete_entry_button`, `show_password_button`, `copy_password_button`, and `copy_username_button`.
+
+
+The event **category** is `item_view.interaction` and the event is recorded as `item_view.click`.
+
+The **method** is `click`.
+
+Example event registration:
+
+```javascript
+Services.telemetry.registerEvents("item_view.interaction", {
+  "item_view.click": {
+    methods: ["click"],
+    objects: ["edit_entry_button",
+		"delete_entry_button",
+        "show_password_button",
+        "copy_password_button",
+        "copy_username_button"],
+    extra: {"fxa_uid": uid}
+  }
+});
+```
+Example of recording a click to show password event:
+
+```javascript
+Services.telemetry.recordEvent("item_view.interaction",
+                    "click",
+                    "show_password_button",
+                    null,
+                    {"fxa_uid": uid});
+```
+
+### Events Related to Interactions in the Item Editors
+These events record actions users take in the item editors. There are separate views for editing existing items and for adding new entries, though the fields that exist in each are similar. Thus we have two **categories** of events: `new_item.interaction` and `edit_item.interaction`.
+
+For events related to the submission of actual item information, see the next section. The item editors have 5 fields: `title_field`, `origin_field` (URL for the credential), `username_field`, `password_field`, and `notes_field`. There are also three buttons: `save_entry_button`, `cancel_button`, and `toggle_password_button`. The latter toggles the visibility of characters in the password field.  
+
+
+The **methods** are `click` and `focus`.
+
+Example event registration (Substitute `new_item` with `edit_item` to register events for the existing item editor):
+
+```javascript
+Services.telemetry.registerEvents("new_item.interaction", {
+  "item_editor.click": {
+    methods: ["click", "focus"],
+    objects: ["title_field",
+        "origin_field",
+        "username_field",
+        "password_field",
+        "notes_field",
+        "save_entry_button",
```

Comment:
Correct me if I'm mistaken, but focus events for buttons do not happen if the mouse (or touch gesture) is used to activate a button.

Assuming the above statement is true, Is it still worth tracking "focus" events for buttons?

Reply:
I removed the focus events from the doc